blocks/blog-list/blog-list.js

```diff
@@ -26,13 +26,14 @@ export async function renderFilterCard(post) {
   cardText.append(
     span({ class: 'card-date' }, publicationDate),
     div({ class: 'card-cta' },
```

Comment:
I think the description should be outside of the `card-cta` div. cta, or Call To Action is the `Read More` button. A description is not a call to action, is information on the card, same as the publication date, in my opinion.

Reply:
To put it differently, the card-cta div, should be on the "else" branch of the ternary operator, not as a parent for the card description.